website/content/en/v1.0/upgrading/v1-migration.md

```diff
@@ -21,6 +21,10 @@ Use the [compatibility matrix]({{<ref "compatibility#compatibility-matrix">}}) t
 Karpenter `v1.0` is a major release and contains a number of breaking changes.
 The following section will highlight some of the major breaking changes, but you should review the full [changelog]({{<ref "#changelog">}}) before proceeding with the upgrade.
 
+### Forceful Expiration
```

Comment:
Other subheadings in this section have are h4 to keep the side navigation bar clean.
```suggestion
#### Forceful Expiration
```